README.md

```diff
@@ -447,6 +448,29 @@ module "setup_tfstate" {
 | s3_bucket_arn  | string | The ARN of the s3 bucket  |
 | s3_bucket_name | string | The name of the s3 bucket |
 
+# Outbound proxy VPC
+
+Whenever you need to talk to APIs which use IP based whitelisting, this is
+the module to create the required setup with. It requires an eip/elastic ip
+and it spits out a vpc\_config which can be attached to a lambda function. The
```

Comment:
Just curious, is there a reason to use `vpc\_config` instead of just `vpc_config`?

Reply:
my ide crying that _ is used for underlining in md and there is no closing for it... :D but will replace it. Does not matter that much